tests/integration/common/auth/test_auth.py

```diff
@@ -358,17 +353,72 @@ def test_platform_admin_remove_all_other_roles(self, anonymous_client, factories
                     "preferred_username": "test.member@communities.gov.uk",
                     "name": "SSO User",
                     "roles": ["FSD_ADMIN"],
-                    "sub": "someStringValue",
+                    "sub": "wer234",
                 }
             }
 
             response = anonymous_client.get(url_for("auth.sso_get_token"), follow_redirects=True)
-            updated_user = db_session.scalar(select(User).where(User.azure_ad_subject_id == "someStringValue"))
+            updated_user = db_session.scalar(select(User).where(User.azure_ad_subject_id == "wer234"))
             assert AuthorisationHelper.is_platform_admin(updated_user) is True
             assert db_session.scalar(select(func.count()).select_from(UserRole)) == 1
 
         assert response.status_code == 200
 
+    def test_grant_member_with_valid_invites_first_login(self, anonymous_client, factories, db_session):
+        with patch("app.common.auth.build_msal_app") as mock_build_msal_app:
```

Comment:
optional nit: a small pre-condition check that the user is anonymous / has no roles here might be nice?